pkg/compile/compile.go

```diff
@@ -31,6 +31,9 @@ var badRules = map[string]bool{
 	"SIGNATURE_BASE_Reconcommands_In_File":                true,
 	"SIGNATURE_BASE_Apt_CN_Tetrisplugins_JS":              true,
 	"CAPE_Sparkrat":                                       true,
+	"SECUINFRA_SUSP_Powershell_Base64_Decode":             true,
+	// Rely on our first-party UPX rule
```

Comment:
Either remove the comment or add it to the above base64 decoding one. Both have internal duplicates.

Reply:
Updated in [`ec3ea08` (#694)](https://github.com/chainguard-dev/malcontent/pull/694/commits/ec3ea08e526c90802b024fb0bb349cc4c070dc7a).